MobileCenter/MobileCenter/MSMobileCenter.m

```diff
@@ -187,9 +187,11 @@ - (void)start:(NSString *)appSecret withServices:(NSArray<Class> *)services {
       NSMutableArray<NSString*>* servicesNames = [NSMutableArray arrayWithCapacity:sortedServices.count];
 
       for (Class service in sortedServices) {
-        [servicesNames addObject:[self startService:service].storageKey];
+        if ([self startService:service]) {
+          [servicesNames addObject:[service logTag]];
```

Comment:
The value of `storageKey` will be changed by @TroubleMakerBen when we have a bucket-per-group thing. Please feel free to add serviceName in the same level of storageKey and implement it in each service class, returns kMSServiceName.